Applications/WebEditor.qmd

```diff
@@ -63,48 +63,96 @@ Based on their applications user can build their model by simple drag and drop m
 
 1. Load Collection 
    
-    In order to load the required collection make sure it is available by searching in the sidebar. Once you find your collection you can simply drag and drop it for carrying out further actions. In the following clip, you can observe the sequential actions are taken to accomplish the following tasks: choosing a collection of interest, defining the spatial and temporal boundaries, and filtering the necessary bands for subsequent processing. Specifically, for calculating the NDVI, the Red band (B04) and Near-Infrared (NIR) band (B08) have been selected.
+    a. To start a task within the openEO web editor, your first step involves ensuring you have the necessary dataset for the analysis. Thus, you must ensure the required dataset exists within the openEO database. You can do this by exploring/searching the list of available datasets from the sidebar of the interface.
+    
+    b. In our example, we want to calculate the Normalized Difference Vegetation Index (NDVI), so the Sentinel 2 L2A collection will be used. Once you find this collection, the next step is to drag and drop it into the Process Editor for further operations.
+        
+    ![](_images/webeditor/drag_collection.png)
+
+
+    Now, once you load collection, there are several parameters you need to specify, including the area of interest, the temporal extent, and the selection of bands. When you click on the collection, a window pops up, where you can define the necessary parameters for subsequent processing. 
+    
+    c. To specify the area of interest, you can choose between generating a bounding box or importing your spatial extent by dragging and dropping GeoJSON or KML files onto the map view. Here, we've provided an example involving a bounding box. There is also an option to "No filter", that will include all the data in the datacube.
```

Comment:
No bounding box filter is generally a bad idea for novice users (the ones that use a wizard). Instead we should point out that the user should start with a sufficiently small boundingbox with respect to the resolution of the data.

Reply:
updated the sentence